101-default-shared-dashboard/azuredeploy.json

```diff
@@ -0,0 +1,87 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "dashboardName": { "type": "string" }
+  },
+  "variables": {
+  },
+  "resources": [
+    {
+      "properties": {
```

Comment:
move the properties node to the bottom of the resource

Reply:
done